online.hatsune_miku.tachidesk-vaadinui.desktop

```
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Name=Tachidesk VaadinUI
+Exec=/usr/bin/flatpak run online.hatsune_miku.tachidesk-vaadinui
```

```suggestion
Exec=vaaui.sh
```

Flatpak will rewrite it for you

---

I've tried that locally, but it always said that vaaui.sh couldn't be found when using it this way

---

not sure what you call be "trying this locally", but the suggestion IS correct.

---

> not sure what you call be "trying this locally", but the suggestion IS correct.

Building and running the app locally. But since you people most definitely know more than me when it comes to flatpaks, I'll just change it

---

If you try to use that desktop file to launch the flatpak, it won't work. Flatpak REWRITE it properly (as mentionned above) when exporting because this one is INSIDE the sandbox. Your testing was incorrect. The simple rule is the Exec line in the .desktop MUST match the the `command` (minus the arguments substitutions)